File: core/Core/Models/CustomerAddress.cs
using System;
using System.Collections.Generic;

namespace Core.Models;

public partial class CustomerAddress
{
    public int AddressId { get; set; }

    public int? CustomerId { get; set; }

    public string? Country { get; set; }

    public string? City { get; set; }

    public string? Street { get; set; }

    public string? ZipCode { get; set; }

    public bool? IsPrimary { get; set; }

    public virtual Customer? Customer { get; set; }
}

File: analytic/domain/customer/customer_address.py
class CustomerAddress:
    def __init__(self,
                 address_id: int,
                 customer_id: int | None,
                 country: str | None,
                 city: str | None,
                 street: str | None,
                 zip_code: str | None,
                 is_primary: bool | None):
        self.address_id = address_id
        self.customer_id = customer_id
        self.country = country
        self.city = city
        self.street = street
        self.zip_code = zip_code
        self.is_primary = is_primary
